doc/tutorials/basic/index.md

```diff
@@ -0,0 +1,332 @@
+:::
+
+:::{grid-item-card} {octicon}`screen-full;2.5em;sd-mr-1` Control the Size
+:link: size
+:link-type: doc
+
+We will discover how sizing works in Panel, exploring the differences between inherent sizing, fixed sizing and responsive sizing. Finally we will cover responsive layouts.
+:::
+
+:::{grid-item-card} {octicon}`arrow-switch;2.5em;sd-mr-1` Align Content
+:link: align
+:link-type: doc
+
+We will align our wind turbine images perfectly using `align`, `margin` and `Spacer`s.
+:::
+
+:::{grid-item-card} {octicon}`file;2.5em;sd-mr-1` Build a Report
+:link: build_report
+:link-type: doc
+
+Construct a static Wind Turbine Report, exportable to HTML for efficient distribution to management.
+:::
+
+::::
+
+## Handle User Input
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`person;2.5em;sd-mr-1` Accept Inputs with Widgets
+:link: widgets
+:link-type: doc
+
+Elevate our app's functionality by accepting user input via widgets. Add nice tooltips with explanations to our users or disable widgets for security reasons.
+:::
+
+:::{grid-item-card} {octicon}`arrow-switch;2.5em;sd-mr-1` React to User Input
+:link: pn_bind
+:link-type: doc
+
+React to user input by leveraging `pn.bind`, enabling our users to foster a deeper understanding of wind turbine data through exploration.
+:::
+
+:::{grid-item-card} {octicon}`list-ordered;2.5em;sd-mr-1` Handle State
+:link: state
+:link-type: doc
+
+Be able to build larger and more complex apps by defining and maintaining state via `pn.rx`.
+:::
+
+:::{grid-item-card} {octicon}`checklist;2.5em;sd-mr-1` Build a Todo App
+:link: build_todo
+:link-type: doc
+
+Enable our wind turbine technicians to keep track of their tasks.
+:::
+
+::::
+
+## Show Activity
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`hourglass;2.5em;sd-mr-1` Display Activity with Indicators
+:link: indicators_activity
+:link-type: doc
+
+Show activity with indicators just as rotating blades show the activity of wind turbines.
+:::
+
+:::{grid-item-card} {octicon}`versions;2.5em;sd-mr-1` Update Progressively
+:link: progressive_layouts
+:link-type: doc
+
+Unleash the power of generators and reactive expressions in our Wind Turbine Apps!
+:::
+
+:::{grid-item-card} {octicon}`image;2.5em;sd-mr-1` Build an Image Classifier
+:link: build_image_classifier
+:link-type: doc
+
+We will build an image classifier to identify wind turbine images.
+:::
+
+::::
+
+## Improve Performance
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`versions;2.5em;sd-mr-1` Add Caching
+:link: caching
+:link-type: doc
+
+Caching allows our apps to store and reuse valuable computations, reducing the energy required for calculations and making our apps run faster and smoother.
+:::
+
+:::{grid-item-card} {octicon}`kebab-horizontal;2.5em;sd-mr-1` Defer the Load
+:link: defer_load
+:link-type: doc
+
+Deferring the load will improve the performance perceived by the user
+:::
+
+::::
+
+## Periodic Updates
```

Comment:
A streaming dashboard can be done with `pn.state.add_periodic_callback` or (`async`) generators.

- Should we include a streaming dashboard example?
- How much should we explain *basic *users about task scheduling? There is also global tasks etc.
- How much should we explain *basic* users about life cycle hooks like `pn.state.onload`?

Maybe this section is really about task scheduling and life cycle hooks leading up to a streaming dashboard?